app/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "showCredits": {
       "commandName": "Project",
-      "commandLineArgs": "Credits List"
+      "commandLineArgs": "Progress List"
```

Comment:
I'm approving this for now, but we should avoid using this approach moving forward. I'm in favour of using the `.vscode/launch.json` file See #216

Reply:
Yeah I'm using Visual studio 22 which everyone recommended.